roles/README.adoc

```diff
@@ -1,63 +1,36 @@
 = Roles Good Practices for Ansible
 
-NOTE: this section has been imported "as-is" from the https://github.com/oasis-roles/meta_standards[OASIS metastandards repository] and still requires re-formatting to fit the overall structure.
+NOTE: this section has been rewritten, using https://github.com/oasis-roles/meta_standards[OASIS metastandards repository] and https://docs.google.com/document/d/1jZUsman8IDSvDt-s6loJ3Y4uZsyse3CB-xzM2DCYaHQ/edit[Ansible Good Practice Recommendations] as a starting place. If you have anything to add or review, please comment.
```

Comment:
The google doc isn't public. Does ist still make sense to link to it?

Reply:
> The google doc isn't public. Does ist still make sense to link to it?

No.  This should be omitted.  And - that google doc should be updated to point to this public doc instead.